code/lib/cli/src/js-package-manager/Yarn1Proxy.ts

```diff
@@ -16,19 +16,6 @@ export class Yarn1Proxy extends JsPackageManager {
     return `yarn ${command}`;
   }
 
-  setRegistryURL(url: string) {
-    if (url) {
-      this.executeCommand('yarn', ['config', 'set', 'npmRegistryServer', url]);
```

Comment:
couldn't the change be here instead? `npmRegistryServer` -> `registry`

Reply:
@yannbf No, it literally uses the `npm` setting AFAICT

Reply:
Alrighty